caasp-environment.yaml

```diff
@@ -1,9 +1,9 @@
 ---
 parameters:
   root_password: linux
-  admin_flavor: m1.large
-  master_flavor: m1.large
-  worker_flavor: m1.large
+  admin_flavor: m1.medium
```

Comment:
This flavor is too small for CaaSP nodes :/

Reply:
done